composer.json

```diff
@@ -87,13 +87,13 @@
         "dist/docker/mysql",
         "docker/mysql"
       ],
-      [
-        "dist/docker/global.php",
-        "docker/global.php.dist"
-      ],
       [
         "dist/docker/README.md",
         "docker/README.md"
+      ],
+      [
+        "dist/docker-sync.yml",
+        "docker-sync.yml"
```

Comment:
Is docker-sync work out of the box or require some additional installation steps?

Reply:
This is default config to make it work out of the box

Reply:
Looks like `docker-sync` requires some additional installation https://docker-sync.readthedocs.io/en/latest/getting-started/installation.htm need to add this information to devdocs.

Reply:
Sure, as it's external tool